package.json

```diff
@@ -1,38 +1,62 @@
 {
   "name": "epam_reactjs_mentoring",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.1.1",
-    "@testing-library/user-event": "^13.5.0",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-scripts": "5.0.1",
-    "web-vitals": "^2.1.4"
-  },
+  "version": "1.0.0",
+  "description": "A single page app for a movie DB",
+  "main": "src/index.js",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "webpack serve --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js",
+    "deploy": "npm run build && gh-pages -d dist",
+    "eslint": "eslint .",
+    "eslint:fix": "eslint ./src --fix",
+    "pretty": "prettier --check .",
+    "pretty:fix": "prettier --write .",
+    "lint": "npm run eslint && npm run pretty",
```

Comment:
Let's go a step further this time and add a pre-push hook with [Husky](https://typicode.github.io/husky/#/)